### PR TITLE
✨ url change

### LINKED ISSUE #40: Change meta tags to point to the new URL
Trap was moved to https://trap.vik.ink/ but some links refer to https://trap.ltd/

### PR DESCRIPTION
**Pull request by Hladikes**

### CODE REVIEW

File: index.tsx [hunk 1]
```diff
@@ -41,7 +41,7 @@ serve(async (request: Request) => {
           <div class="w-full p-7 md:w-3/4 2xl:w-3/5">
             <h1 class="text-white text-5xl sm:text-6xl mt-7 sm:mt-12">Trapcards</h1>
             <p class="text-white font-mono -mx-7 sm:mx-0 my-7 p-4 px-8 sm:px-4 bg-white/5 text-xl whitespace-nowrap overflow-x-auto">
-              <span>https://trap.ltd/</span><span class="text-yellow-500">[trapcard name]</span>
+              <span>https://trap.vik.ink/</span><span class="text-yellow-500">[trapcard name]</span>
```
Comment (vikiival) [inline index.tsx]: Maybe something like `window.location.origin` ?

**Review by vikiival — COMMENTED** [review]
Just more reusable ?

**Comment by Hladikes** [comment]
@vikiival I'm on it

**Comment by Hladikes** [comment]
image

**Review by vikiival — APPROVED** [review]
Love it!

**Pull request merged**